diff-viewer/src/styles.css

```
@@ -0,0 +1,607 @@
+  --cmux-diff-ui-line-height: 16px;
+  --cmux-diff-code-font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", monospace;
+  --cmux-diff-font-size: 10px;
+  --cmux-diff-line-height: 20px;
+  --cmux-diff-bg: var(--cmux-diff-bg-light);
+  --cmux-diff-fg: var(--cmux-diff-fg-light);
+  --cmux-diff-border: color-mix(in lab, var(--cmux-diff-fg) 12%, transparent);
+  --cmux-diff-sidebar-bg: transparent;
+  --cmux-diff-muted-bg: color-mix(in lab, var(--cmux-diff-fg) 8%, transparent);
+  --cmux-diff-hover-bg: color-mix(in lab, var(--cmux-diff-fg) 10%, transparent);
+  --cmux-diff-accent: light-dark(#0a84ff, #7ab7ff);
+  background: transparent;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)